main/SAPI.c

```diff
@@ -231,6 +236,11 @@ static void sapi_read_post_data(void)
 	}
 }
 
+SAPI_API void sapi_read_post_data(void)
```

Comment:
Why do you need to export it?

Reply:
Likely from an old iteration, thanks for noticing.